__tests__/app.test.js

```diff
@@ -159,3 +159,37 @@ describe("Users", () => {
     });
   });
 });
+describe("Comments", () => {
+  describe("getCommentsById", () => {
+    test("status: 200 and resonds with an array of comments objects relating to the article_id", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          comments.forEach((comment) => {
+            expect(comment.comment_id).toEqual(expect.any(Number));
+            expect(comment.author).toEqual(expect.any(String));
+            expect(comment.body).toEqual(expect.any(String));
+            expect(comment.created_at).toEqual(expect.any(String));
+            expect(comment.votes).toEqual(expect.any(Number));
+          });
+        });
+    });
+    test("status: 400 when given an invalid address", () => {
+      return request(app)
+        .get("/api/articles/one/comments")
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
+    });
+    test("status: 404 when given an valid address but no data", () => {
+      return request(app)
+        .get("/api/articles/100/comments")
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe("Article_id doesn't exist");
+        });
```

Comment:
Can you please add a test for an article with no comments coming back with a status 200 and empty array